packages/nextjs/rollup.config.js

```diff
@@ -37,8 +37,6 @@ export default {
     commonjs(),
     copy({
       targets: [
-        { src: 'build/copy-config-files.js*', dest: 'dist' },
```

Comment:
The issue is here. The Typescript built version is transpiled to esnext, not commonjs.